docs/configuration.rst

```diff
@@ -654,9 +689,27 @@ Map Options
     is required. If you want a high detail, use texture size 16, but texture 
     size 12 looks still good and is faster to render.
 
-    Mapcrafter's pre-rendered textures include sizes 16 and 12. If you want to 
+    Mapcrafter's pre-rendered spritemaps include sizes 16 and 12. If you want to 
     use other sizes, or custom resource packs you will need to generate them using 
-    `blockcrafter <https://github.com/mapcrafter/blockcrafter>`_.
+    `Blockcrafter <https://github.com/mapcrafter/blockcrafter>`_.
+
+
+**Block Directory** ``block_dir = <directory>``
+
+    **Default:** ``data/blocks``
+
+    This is the folder where spritemaps files are located. You can set this
+    to use textures from a custom resource pack. 
+
+    The standard Minecraft spritemaps are included with Mapcrafter, if you want to
```

Comment:
Maybe we should specify for which configurations spritemaps are delivered exactly, e.g. isometric/topdown/side 12px and 16px.

Reply:
I've got a separate branch for a new Mapcrafter page, so I'll include the more verbose list of out-of-the-box configurations there. Otherwise, I think "everything in 16px and 12px" is a better shorthand.